test-infra/airflow/Dockerfile.prow

```diff
@@ -12,19 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-IMG = gcr.io/mlkube-testing/builder
-TAG = $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)
-
-all: build
-
-build:
-	@echo {\"image\": \"$(IMG):$(TAG)\"} > version.json
-	docker build -t $(IMG):$(TAG) .
-	docker tag $(IMG):$(TAG) $(IMG):latest
-	@echo Built $(IMG):$(TAG) and tagged with latest
-	rm -f version.json
-
-push: build
-	gcloud docker -- push $(IMG):$(TAG)
-	gcloud docker -- push $(IMG):latest
-	@echo Pushed $(IMG) with :latest and :$(TAG) tags
+# This Dockerfile is used by ProwJobs to launch our Airflow jobs.
+#
+# TODO(jlewi) With Docker v17.0 we should just be able to use Docker
+# build args to specify the base image.
+FROM gcr.io/mlkube-testing/airflow:v20171119-39818ad-dirty-13370f
```

Comment:
will this repo name be changed?

Reply:
mlkube-testing is a GCP project. We could change it but its not high priority right now. The name of our test project isn't really user facing.